src/lammpsio/snapshot.py

```diff
@@ -48,6 +48,9 @@ def __init__(self, N, box, step=None):
         self._typeid = None
         self._charge = None
         self._mass = None
+        self._bonds = None
+        self.Nbonds = None
+        self.bondtype = None
```

Comment:
With the way other things are named, I think it should be like this:
```suggestion
        self._bond = None
        self.num_bonds = None
        self.bond_typeid = None
```
Alternatively, we should add a class like `ParticleConnections` (probably need a better name), which we can then reuse for all the bonded data like:
```
self._bonds = ParticleConnections(num_bonds, 2)

@property
def bonds(self):
    return self._bonds
```
then later on:
```
snap.bonds.N
snap.bonds.id
snap.bonds.typeid
snap.bonds[i]
```
This seems pretty slick / clean to me (and is similar to how GSD works), so we can discuss how to do that if you want to try it.